pkg/agent/openflow/pipeline.go

```diff
@@ -1652,6 +1654,28 @@ func (c *client) conjunctionActionDenyFlow(conjunctionID uint32, table binding.T
 		Done()
 }
 
+func (c *client) conjunctionActionPassFlow(conjunctionID uint32, table binding.Table, priority *uint16, disposition uint32, enableLogging bool) binding.Flow {
+	ofPriority := *priority
+	nextTable := IngressRuleTable
+	tableID := table.GetID()
+	if _, ok := egressTables[tableID]; ok {
+		nextTable = EgressRuleTable
+	}
+	flowBuilder := table.BuildFlow(ofPriority).
+		MatchConjID(conjunctionID).
+		Action().LoadRegMark(DisposttionPassRegMark)
+	if enableLogging {
+		customReason := CustomReasonLogging
+		flowBuilder = flowBuilder.
+			Action().LoadToRegField(APDispositionField, disposition).
```

Comment:
L1666 and L1670 are basically doing the same thing. 
`Drop` and `Reject` use the same func `conjunctionActionDenyFlow`, so they need `disposition` param. Since `Pass` has its own func `conjunctionActionPassFlow` and disposition reg is just used in logging (@qiyueyao right?), I think you can remove `disposition` param and just use `Action().LoadRegMark(DispositionPassRegMark)` when logging is enabled. 
BTW L1666 also has a typo.

Reply:
Makes sense to me.

Reply:
Makes sense to me.

Reply:
Thanks, fixed

Reply:
Sorry I didn't express myself clearly. I mean if disposition reg is only used for logging purpose, then we should load this reg inside `if enableLogging {}`.

Reply:
Done